### DISCUSSION TITLE
Any chance to add Notion Tag Picker Again

### DISCUSSION THREAD

**Discussion by daddasoft** [mantinedev/mantine#1629] [Q&A] [answered]
<img src="https://user-images.githubusercontent.com/5988991/135754339-cabed292-27d3-4444-ac15-7eb8b2467b45.gif">

**Comment by rtivital** [comment, marked as answer]
Maybe, cannot promise anything

**Reply by daddasoft** [reply]
we hope  that happen it was a great component
